app/views/notFoundError.scala.html

```diff
@@ -8,15 +8,13 @@
     @defining(play.core.PlayVersion.current) { version =>
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
-                <h1 class="govuk-heading-xl">404 - Page not found</h1>
-                <p>This is a placeholder 404 error page</p>
-                <span id="general-error" class="govuk-error-message">
-                    @errorMessage
-                </span>
+                <h1 class="govuk-heading-l">Page not found</h1>
+                <p class="govuk-body">If you typed a web address, check it is correct.</p>
+                <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
+                <p class="govuk-body">
+                  If the web address is correct or you selected a link or button, <a class="govuk-link" href="#">contact The National Archives</a> if you need to speak to someone about your transfers.
```

Comment:
Where should this link go to? It currently just goes to the same page.

Reply:
I would imagine this would go to Claire's email address as she deals with transfers, but if we want this to link to a TDR help desk we could also consider that. I wasn't too sure about this when I was designing it too.

Reply:
Could you find out where it should go to and update it?

Reply:
I spoke with Kirsten and Sarra regarding this and they suggested that we have two links on this page, one to let the user return back to the home page and the other to let the user contact a system admin/transfer advisor, essentially it should go to a “TDR help desk”.

At the moment we haven’t got that set up so we’ll need to think about where this link would go for the time being.

Reply:
Thanks for checking with them. Could you create a card so that we can update the contact link when we've created the mailing list?

Reply:
No worries, I've added that here: https://national-archives.atlassian.net/browse/TDR-601